kymatio/scattering1d/backend/jax_backend.py

```diff
@@ -0,0 +1,5 @@
+
+from .numpy_backend import NumpyBackend1D
+import jax.numpy 
+
+backend = NumpyBackend1D(jax.numpy)
```

Comment:
This thing, according to what I saw above will still have `self.name` be `numpy`